src/components/slider/context/SliderGroup.tsx

```diff
@@ -11,40 +12,24 @@ interface SliderGroupProps {
   children?: React.ReactNode;
 }
 
-interface SliderGroupState {
-  value: tinycolor.ColorFormats.HSLA;
-}
-
-export default class SliderGroup extends Component<SliderGroupProps, SliderGroupState> {
-  static displayName = 'IGNORE';
+const SliderGroup = (props: SliderGroupProps) => {
+  const {color, onValueChange, children} = props;
+  const [value, setValue] = useState(Colors.getHSL(color));
 
-  constructor(props: SliderGroupProps) {
-    super(props);
+  const valueSetter = useCallback((value: tinycolor.ColorFormats.HSLA) => {
+    setValue(value);
+    onValueChange?.(Colors.getHexString(value));
+  },
+  [onValueChange]);
 
-    this.state = {
-      value: Colors.getHSL(props.color)
-    };
-  }
+  const providerValue = useMemo(() => ({value, setValue: valueSetter}), [value, valueSetter]);
```

Comment:
Naming - 'contextProviderValue' - use a more descriptive name as "provider" is not clear enough